src/app/lists/CommentBox.tsx

```diff
@@ -0,0 +1,128 @@
+"use client";
+
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import supabase from "@/supabase";
+import Button from "@mui/material/Button";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+interface CommentProps {
+    originalComment: string;
+}
+
+const HeaderAndButtonContainer = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+    min-width: 8rem;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: space-between;
+    @media (min-width: 800px) {
+        width: 15%;
+    }
+`;
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    background-color: ${(props) => props.theme.main.background[2]};
+    border-radius: 1rem;
+    margin: 0.5rem;
+    padding: 0.5rem;
+`;
+
+const CommentBoxContainer = styled.div`
+    width: 100%;
+    margin: 1rem;
+    min-width: 15rem;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    @media (min-width: 800px) {
+        width: 75%;
+    }
+`;
+
+const ButtonContainer = styled.div`
+    display: flex;
+    width: 100%;
+    height: 90%;
+    gap: 1rem;
+    max-width: 15rem;
+    justify-content: space-around;
+    @media (min-width: 800px) {
+        flex-direction: column;
+        align-items: center;
+        max-width: 25rem;
+    }
+`;
+
+const ErrorText = styled.p`
+    color: ${(props) => props.theme.error};
+    margin: 1rem 0 0;
+    font-size: 0.8rem;
+`;
+
+const SuccessText = styled.p`
+    color: ${(props) => props.theme.primary.background[3]};
+    margin: 1rem 0 0;
+    font-size: 0.8rem;
+`;
+
+const CommentContainer: React.FC<CommentProps> = (props) => {
```

Comment:
I prefer converting `props` to `{originalComment}`, unless there's _lots_ of variables!